### PR TITLE
For additional

### PR DESCRIPTION
**Pull request by AkashiX**
Make up more information for.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,69 @@ exports.userservice = {
 }
 ```
 
+## Detail Example
+What author mean that is conveys a concept to you of 'userservice', I guess.
+You can do as follow:
+
+config/config.default.js
+```js
+
```
Comment (atian25) [inline README.md]: put `// config/config.default.js` in code block

File: README.md [hunk 2]
```diff
@@ -71,6 +71,69 @@ exports.userservice = {
...
+    // Mount middleware
+    // Your must do this, or you can't work well
+    config.middleware = [
+        'userservice'
```
Comment (atian25) [inline README.md]: 2 spaces, same to below code

File: README.md [hunk 3]
```diff
@@ -71,6 +71,69 @@ exports.userservice = {
...
+    ];
+
+     // Mount retrived user datas
+     config.userservice = {
+        service: {
+            getUserId(ctx) {
+                return ctx.user && ctx.user.uid;
+            },
+            * getUser(ctx) {
+                console.log('ctx', ctx);
```
Comment (atian25) [inline README.md]: remove

File: README.md [hunk 4]
```diff
@@ -71,6 +71,69 @@ exports.userservice = {
...
 ```
 
+## Detail Example
+What author mean that is conveys a concept to you of 'userservice', I guess.
+You can do as follow:
+
+config/config.default.js
+```js
+
+    // Mount middleware
```
Comment (atian25) [inline README.md]: no indent

File: README.md (hunk 1 above)
Comment (AkashiX) [inline README.md]: Thanx, I had changed.

Comment (atian25) [inline README.md]: mv to code block, under `js`

**Comment by shaoshuai0102** [comment]
对用户没有太大帮助，只是重复了 repo 里的代码和配置，这个自己翻代码就可以了

**Pull request closed without merging**